proto/serverless/common/v1/common.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
```

Comment:
This is the naming standard that most protobuf packages use. I don't want to make changes to this anymore, Roberto and others are already starting to use these internally.

Reply:
But current names don't make much sense. Why repeat `common` token? Can't we make it right?

Reply:
I'm not understanding what the confusion here is. The package name is `serverless.common.v1`. The files within that folder just map into that

Reply:
This comment refers to filename not package name

Reply:
Filenames here do not map to package names. See top comment.

Reply:
Yes it does. The package is `serverless/common/v1` not `serverless/common/v1/common`.  The file name itself does not matter, it just affects what gets generated. The directory is the package not the file.

Reply:
> The directory is the package not the file.

Why directory and not file, If it's file that represents the package?

Reply:
That's just the convention we use to support the most generalized approach. We have the ability in this approach to have a common package that will contain many common properties. The name `common.proto` is just due to that being all we have. We can rename it to something else if it really bugs you but I am not naming it `v1.proto`

Reply:
`serverless/common/v1.proto` will exactly reflect the package name. Why it's a no go?

Reply:
This may be moot, we don't need a lot of what is in that file anymore. Let me restructure it a bit with the changes we are making to having global tags.

Reply:
@medikoo I've gone ahead and removed the common package as we do not need it right now. I just put the global tag structure in the `tags` package.

Reply:
Ok, still my comment was about file naming convention of all .proto files

Currently PR proposes:

```
serverless.instrumentation.tags.v1                    <- package name
serverless/instrumentation/tags/v1/tags.proto         <- filename
```

My question is why not:

```
serverless.instrumentation.tags.v1            <- package name
serverless/instrumentation/tags/v1.proto      <- filename
```

The latter seems to me more logical

Reply:
That only is more logical if we will never want to have more than one file in the tags package.

Say for example we add containers so we want to split the single `tags.proto` into a `lambda_tags.proto` and `container_tags.proto` we could not do that in your convention, we would have to create multiple packages then vs a single tags package. 

You are trying to attach the convention that a package is a file, which is not the case, that is an anti-pattern we should not follow just for directory readability.

Reply:
Ok, so according to your comment, I think file should be named either:
1.  `serverless/instrumentation/tags/v1/index.proto` - _index_ exposes whole package, and if it's content is to be split into multiple files it can just import them, and still maintain contract of exposing whole package (individual schemas can be updated to import just portions they need)
2.  `serverless/instrumentation/tags/v1/aws_lambda.proto` - This actually creates a room for putting _container_ tags into other file. If it stays as `tags.proto`, such addition will require rename of this file, which signals this file is not named right from the start

Reply:
>  index exposes whole package, and if it's content is to be split into multiple files it can just import them, and still maintain contract of exposing whole package

That isn't how imports/exports would work. we get no benefit from this. 

I'm fine to go with option 2.